packages/twenty-server/src/utils/generate-front-config.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export function generateFrontConfig(): void {
+  const configObject = {
+    window: {
+      _env_: {
+        REACT_APP_SERVER_BASE_URL:
+          process.env.REACT_APP_SERVER_BASE_URL || 'http://localhost:3000',
```

Comment:
Could you use the environmentService instead of process.env?
Also on the backend side we have ` SERVER_URL` variable already so it could be  REACT_APP_SERVER_BASE_URL: process.env.SERVER_URL;
If `SERVER_URL` isn't set, then I would leave it empty. Localhost:3000 isn't the best fallback, we have a better fallback based on the browser's window.location (which we can't do on the backend side) in config/index.ts

Thanks!